### PR TITLE
ACM-8980 Fix API doc from suggestions made by the doc team

### PR DESCRIPTION
**Pull request by fxiang1**
https://issues.redhat.com/browse/ACM-8980

- Update API description as requested by the doc team

### CODE REVIEW

**Comment by fxiang1** [comment]
/assign @oafischer 

Oliver, can you review this again? The other PR https://github.com/stolostron/cluster-curator-controller/pull/190 got merged before I can make the suggested changes. Thanks!

File: deploy/crd/cluster.open-cluster-management.io_clustercurators.yaml [hunk 1]
```diff
@@ -20,9 +20,9 @@ spec:
     schema:
       openAPIV3Schema:
         description: ClusterCurator is the custom resource for the clustercurators
-          API. This kind allows for Ansible prehook and posthook jobs to be run prior
-          to Hive or HyperShift provisioning and import of a cluster. Additionally
-          cluster upgrade and destroy operations are supported as well.
+          API. This kind allows you to run Ansible prehook and posthook jobs before
+          provisioning Hive or HyperShift and importing a cluster. Additionally cluster
```
Comment (oafischer) [inline deploy/crd/cluster.open-cluster-management.io_clustercurators.yaml:24]:
```suggestion
          provisioning Hive or HyperShift and importing a cluster. Additionally, cluster
```

**Comment by oafischer** [comment]
Didn't know it would merge automatically, just reviewed again, thanks!

**Review by oafischer — APPROVED** [review]

**Comment by fxiang1** [comment]
Thanks @oafischer !

**Pull request merged**